src/cloud/docker/docker-mftf.md

```diff
@@ -89,6 +89,12 @@ To set up and run MFTF tests in a Cloud Docker environment:
    docker-compose run deploy magento-command setup:config:set  --http-cache-hosts=varnish
    ```
 
+1. Clear the cache.
+
+   ```bash
+   docker-compose run deploy magento-command cache:clean
```

Comment:
All `docker-compose run` must be changed to `docker-compose run --rm` to avoid having orphan containers